docs/components/snackbars.md

```diff
@@ -0,0 +1,238 @@
+<!--docs:
+title: "Material Snackbars"
+layout: detail
+section: components
+excerpt: "Snackbars provide brief messages about app processes at the bottom of the screen."
+iconId: 
+path: /catalog/Snackbars/
+-->
+
+# Snackbars
+
+[Snackbars](https://material.io/components/snackbars) provide brief messages about app processes at the bottom of the screen.
+
+![Snackbars hero image](assets/snackbars/snackbars-hero.png)
+
+## Contents
+
+* [Contents](#contents)
+* [Using snackbars](#using-snackbars)
+* [Snackbars example](#snackbar-example)
+* [Theming snackbars](#theming-example)
+
+## Using snackbars
+
+Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.
+
+### Making snackbars accessible
+
+A SnackBar with an action will not time out when TalkBack or VoiceOver are enabled. This is controlled by [`AccessibilityFeatures.accessibleNavigation`](https://api.flutter.dev/flutter/dart-ui/AccessibilityFeatures/accessibleNavigation.html).
+
+### Anatomy and key properties
+
+The following is an anatomy diagram of a snackbar:
+
+![Snackbar anatomy diagram](assets/snackbars/snackbars-anatomy.png)
+
+1. Text label
+1. Container
+1. Action (optional)
+
+#### Text label properties
+
+| &nbsp; | Property |
+|---|---|
+| **Text label** | pass a Widget to `content`|
+| **Text color** | pass a Widget to `content`|
+| **Typography** | pass a Widget to `content`|
+
+#### Container properties
+
+&nbsp; | Property
+------ | --------- 
+**Color** | `backgroundColor`
+**Elevation** | `elevation`
+**Height** |  N/A
```

Comment:
I think the recommendation is to remove N/A properties

Reply:
done